_common/ImageRandomizer.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Keyman\Site\Common;
+
+class ImageRandomizer {
+    
+    static function randomizer($folder) { 
+        if(!empty($_SERVER['DOCUMENT_ROOT']) && !empty($folder)) {
+          $imgDir = $_SERVER['DOCUMENT_ROOT'] . '/cdn/dev/' . $folder;
```

Comment:
@mcdurdin - I have some big picture questions about the folder hierarchy.
1. Should the /_common/assets/*.png files stay in place, or does a `configure` step copy them to the `/cdn/dev/` folders (which then convert to `/cdn/deploy/`

Reply:
hmm, in a perfect world configure step would copy them, because that gives the cdn cache busting filenames. but it's a lot of rubber band and glue for somewhat limited benefit. I think I am happy with them being referenced straight out of _common.

Reply:
In that case I think the path in line 9  `'/cdn/dev/'` changes to `'/_common/'` (where the logos currently reside on the PR).  Is `$path` set to `_common`?

Reply:
I'm not sure what the `$path` is, @darcywong00? 

This would be the change, is that correct?
```php
$imgDir = $_SERVER['DOCUMENT_ROOT'] . '/_common/' . $folder;
```

Reply:
Yeah, and then $folder would be "assets"